src/Alert/index.js.flow

```diff
@@ -9,14 +9,14 @@ type Type = "info" | "success" | "warning" | "critical";
 
 export type Props = {|
   +type?: Type,
-  +children?: React$Node,
+  +children?: React.Node,
   +title?: Translation,
-  +icon?: React$Element<any> | boolean,
+  +icon?: React.Element<any> | boolean,
```

Comment:
I believe that it's necessary to import `React` into every `.js.flow` file that is using it. If I remember correctly, the flow gives you error otherwise.